src/Components/VimeoVideo.js

```diff
@@ -62,7 +64,7 @@ const VimeoVideo = ({ vimeoId, screenWidth }) => {
   const renderError = () => {
     return (
       <View style={styles.spinnerStyle}>
-        <Text>Oopsie! There was an error loading the video...</Text>
+        <Text> Oopsie! There was an error loading the video...</Text>
```

Comment:
You added a space here, you need to remove it or also add it in the test's `getText`